packages/x-date-pickers/src/locales/euES.ts

```diff
@@ -0,0 +1,87 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+import { TimeViewWithMeridiem } from '../internals/models';
+
+const views: Record<TimeViewWithMeridiem, string> = {
+  hours: 'orduak',
+  minutes: 'minutuak',
+  seconds: 'segunduak',
+  meridiem: 'meridianoa',
+};
+
+const euESPickers: Partial<PickersLocaleText<any>> = {
+  // Calendar navigation
+  previousMonth: 'Azken hilabetea',
+  nextMonth: 'Hurrengo hilabetea',
+
+  // View navigation
+  openPreviousView: 'azken bista ireki',
+  openNextView: 'hurrengo bista ireki',
+  calendarViewSwitchingButtonAriaLabel: (view) =>
+    view === 'year'
+      ? 'urteko bista irekita dago, aldatu egutegi bistara'
+      : 'egutegi bista irekita dago, aldatu urteko bistara',
+
+  // DateRange placeholders
+  start: 'Hasi',
+  end: 'Bukatu',
+
+  // Action bar
+  cancelButtonLabel: 'Utxi',
+  clearButtonLabel: 'Garbitu',
+  okButtonLabel: 'OK',
+  todayButtonLabel: 'Gaur',
+
+  // Toolbar titles
+  datePickerToolbarTitle: 'Data aukeratu',
+  dateTimePickerToolbarTitle: 'Data eta ordua aukeratu',
+  timePickerToolbarTitle: 'Ordua aukeratu',
+  dateRangePickerToolbarTitle: 'Data tartea aukeratu',
+
+  // Clock labels
+  clockLabelText: (view, time, adapter) =>
+    `Aukeratu ${views[view]}. ${
+      time === null
+        ? 'Denbora aukeraketarik gabe'
+        : `Aukeratutako denbora ${adapter.format(time, 'fullTime')} da`
+    }`,
+  hoursClockNumberText: (hours) => `${hours} ordu`,
+  minutesClockNumberText: (minutes) => `${minutes} minutu`,
+  secondsClockNumberText: (seconds) => `${seconds} segundu`,
+
+  // Digital clock labels
+  selectViewText: (view) => `Aukeratu ${views[view]}`,
+
+  // Calendar labels
+  calendarWeekNumberHeaderLabel: 'Astea zenbakia',
+  calendarWeekNumberHeaderText: '#',
+  calendarWeekNumberAriaLabelText: (weekNumber) => `${weekNumber} astea`,
+  calendarWeekNumberText: (weekNumber) => `${weekNumber}`,
+
+  // Open picker labels
+  openDatePickerDialogue: (value, utils) =>
+    value !== null && utils.isValid(value)
+      ? `Aukeratu data, aukeratutako data ${utils.format(value, 'fullDate')} da`
+      : 'Aukeratu data',
+  openTimePickerDialogue: (value, utils) =>
+    value !== null && utils.isValid(value)
+      ? `Aukeratu ordua, aukeratutako ordua ${utils.format(value, 'fullTime')} da`
+      : 'Aukeratu ordua',
+  // fieldClearLabel: 'Clear value',
```

Comment:
Maybe you could provide the translation for this label as well? 🤔 
It would make this locale translation complete. 👏

Reply:
I just discovered that the Spanish translation (which we used as the reference) also lacks that label. Upon closer inspection, I've identified some mistakes, so I will correct the locale for Spanish and update the Basque translations once more. It appears that there might be similar issues in Catalan and other languages (maybe they used Spanish as the reference too, and repeated the mistakes), but I can only assist with Spanish and Basque.

The biggest mistakes are:

```ts
// Current translations (wrong)
clockLabelText: (view, time, adapter) =>
  `Seleccione ${views[view]}. ${
    time === null
      ? 'Sin tiempo seleccionado'
      : `El tiempo seleccionado es ${adapter.format(time, 'fullTime')}`
  }`,
timeTableLabel: 'elige la fecha',
dateTableLabel: 'elige la hora',

// Fixed translations
clockLabelText: (view, time, adapter) =>
  `Seleccione ${views[view]}. ${
    time === null
      ? 'No hay hora seleccionada'
      : `La hora seleccionada es ${adapter.format(time, 'fullTime')}`
  }`,
timeTableLabel: 'elige hora',
dateTableLabel: 'elige fecha',
```

I think there are a few more locales where `timeTableLabel` and `dateTableLabel` have been confused, but I can't assure it.

Once my colleague provides me with the Basque translations I require, I'll change this PR's title and make a few commits fixing this and adding the label you mentioned. :)